react/components/CategoriesHighlights/categoriesHighlights.css

```diff
@@ -0,0 +1,88 @@
+.squared:only-child,
+.rectangular:only-child {
+  display: none;
+}
+
+:global(.vtex-carousel) + .squared,
+:global(.vtex-carousel) + .rectangular {
+  top: -90px;
+}
+
+.squaredCard,
+.rectangularCard {
+  background: rgba(61, 61, 61, 0.8);
+  margin: 2.5px;
+  height: 300px;
+}
+
+.squaredCard {
+  width: 300px;
+}
+
+.rectangularCard {
+  width: 600px;
+}
+
+.squaredCard img,
+.rectangularCard img {
+  object-fit: cover;
+  height: 300px;
+}
+
+.squaredCard img {
+  width: 300px;
+}
+
+.rectangularCard img {
+  width: 600px;
+}
+
+@media only screen and (min-width: 320px) and (max-width: 800px) {
+  :global(.vtex-carousel) + .squared {
+    top: -50px;
+  }
+
+  :global(.vtex-carousel) + .rectangular {
+    top: 0px;
+  }
+
+  .squaredCard,
+  .squaredCard img {
```

Comment:
I not a good idea write css rules to html elements (applying css rules directly to html elements is slower than applying to a class), instead of doing that you could put a class in the `img` element and then write something like `.squaredCard myImageClass`